packages/sanity/src/_internal/cli/commands/backup/downloadBackupCommand.ts

```diff
@@ -0,0 +1,288 @@
+import {createWriteStream, existsSync, mkdirSync} from 'fs'
+import {tmpdir} from 'os'
+import path from 'path'
+import {mkdtemp} from 'node:fs/promises'
+import {randomKey} from '@sanity/block-tools'
+import type {
+  CliCommandArguments,
+  CliCommandContext,
+  CliCommandDefinition,
+  SanityClient,
+} from '@sanity/cli'
+import {absolutify} from '@sanity/util/fs'
+import {isBoolean, isNumber, isString} from 'lodash'
+import prettyMs from 'pretty-ms'
+import {Mutex} from 'async-mutex'
+import {ProgressData} from 'archiver'
+import chooseBackupIdPrompt from '../../actions/backup/chooseBackupIdPrompt'
+import resolveApiClient from '../../actions/backup/resolveApiClient'
+import downloadAsset from '../../actions/backup/downloadAsset'
+import downloadDocument from '../../actions/backup/downloadDocument'
+import newProgress from '../../actions/backup/progressSpinner'
+import {PaginatedGetBackupStream, File} from '../../actions/backup/fetchNextBackupPage'
+import {archiveDir, humanFileSize} from '../../actions/backup/archiveDir'
+import cleanupTmpDir from '../../actions/backup/cleanupTmpDir'
+import {defaultApiVersion} from './backupGroup'
+
+const debug = require('debug')('sanity:backup')
```

Comment:
Not a blocker, but ideally this should be

```suggestion
import createDebug from 'debug'

const debug = createDebug('sanity:backup')
```

Reply:
Fixed it.